.taskcluster.yml

```diff
@@ -295,6 +295,14 @@ tasks:
                                       type: 'directory'
                                       path: '/builds/worker/artifacts'
                                       expires: {$fromNow: '1 year'}
+                                  'public/docker-contexts':
```

Comment:
I'm not sure to follow. What does the decision task expose at this point?

Reply:
The way that building an image with docker worker (either locally or in automation), is that a tarball is created with the docker file, and and any appropriate local files[1]. Previously, this was built in the image-builder task, which had a full checkout, and handled things like (`# %include` and `# %ARG` and `# %include-run-task`). The switch to kaniko moved that logic to the decision task, which exposes the context as artifacts. The decision task was already generating the context, in order to hash it, and the image builder image no longer has any checkout. This also avoid issues like [Bug 1621264](https://bugzilla.mozilla.org/show_bug.cgi?id=1621264) as the context is built in the decision task, which has a specific taskgraph version checked out, rather than the one backed into the image building image.


[1] when running locally, this is the directory passed to `docker build`, with the files listed in `.dockerignore` excluded.